lib/stdlib/src/edlin.erl

```diff
@@ -192,7 +195,8 @@ key_map($\^F, none) -> forward_char;
 key_map($\^H, none) -> backward_delete_char;
 key_map($\t, none) -> tab_expand;
 key_map($\^K, none) -> kill_line;
-key_map($\^L, none) -> redraw_line;
+key_map($\^L, none) -> clear;
+key_map($\^O, none) -> redraw_line;
```

Comment:
Why did you use `^O` for line redraw?

Reply:
I picked one that was not in use and has [no general meaning in the terminal](https://www.makeuseof.com/linux-bash-terminal-shortcuts/) (unless under history search).

Reply:
The reason I ask is that we used ^O in #5924 to trigger opening of an editor to edit the current line.

Reply:
^Oh! I think ^T is also fine, so I will roll with that.

Reply:
Scratch that, ^T is transpose. I think ^N is fine. I will give it a try.

Reply:
^N is the down arrow. :'(

We are pretty much running out of characters here. Given ^L is used mostly for #6260, I wonder if it is worth removing the old feature altogether so we get one character back?

Anyway, I think ^Q is actually fine.